### PR TITLE
Formatted splash page to match landing page text Issue #365

### PR DESCRIPTION
**Pull request by anne27**
Increased the logo size to make it look more prominent.
Also included text to make it consistent across pages.
Please suggest anything you'd like to be changed!
screenshot from 2017-03-05 03-28-11

### CODE REVIEW

File: client/app/layouts/splash.html [hunk 1]
```diff
@@ -1,3 +1,10 @@
+<style>
+md-heading-splash{
+    font-family: 'Cabin Sketch', sans-serif;
+    color: #795548;
+    font-size: 50px;
+  }</style>
+  <!-Can be added to angular-material.css>
```
Comment (tiltec) [inline client/app/layouts/splash.html]: angular-material is not written by us, so we can't change it. I would recommend to put this style in the `app.styl` (supports CSS, but also the simplified [Stylus language](http://stylus-lang.com/). And maybe drop the `md-` prefix to make clear that it's our style and not from Material design :)

What about a structure like this:
```styl
.splash
  heading
    font-family: 'Cabin Sketch'
     // ... styles from splash.html
  // other styles 
```

(Further thought: maybe we should put the splash page style in a `splash.styl` next to `splash.html`... but I guess that's another task)

File: client/app/layouts/splash.html [hunk 2]
```diff
@@ -21,7 +28,7 @@
   <div layout="column" layout-align="center center" flex="noshrink" layout-margin>
     <div layout layout-align="center center">
       <logo></logo>
-      <h1 class="title">Foodsaving Worldwide</h1>
+      <md-heading-splash align="center">Foodsaving<br>Worldwide</md-heading-splash>
```
Comment (nicksellen) [inline client/app/layouts/splash.html]: custom element names are intended for angular directives or components, if its just a normal element then using the most semantic element is best, such as h1

**Comment by anne27** [comment]
Updated splash.html and app.styl.
Now the formatting and the HTML code are in different files. :)

File: client/app/layouts/splash.html [hunk 3]
```diff
@@ -21,7 +21,7 @@
   <div layout="column" layout-align="center center" flex="noshrink" layout-margin>
     <div layout layout-align="center center">
       <logo></logo>
-      <h1 class="title">Foodsaving Worldwide</h1>
+      <heading align="center">Foodsaving<br>Worldwide</heading>
```
Comment (nicksellen) [inline client/app/layouts/splash.html]: heading isn't a proper element either. you can see them at https://developer.mozilla.org/en/docs/Web/HTML/Element. you can pick random words and they will behave like div elements but there should be good reason (like angular directives/components)

Comment (tiltec) [inline client/app/layouts/splash.html]: Agreed - we should reserve custom names for angular. Maybe something for the `contribute.md` (which I didn't have a look inside for quite some time)

Comment (anne27) [inline client/app/layouts/splash.html]: Thanks, that was a great article.  Tere are only two options- <h1-h6> or <header>. <header> might sound a tad ambiguous. I guess <h1> makes more sense here?

File: client/app/app.styl [hunk 4]
```diff
@@ -1,6 +1,11 @@
 .splash
   background-image: url('yunityorg.jpg')
   background-size: cover
+  heading
```
Comment (tiltec) [inline client/app/app.styl]: I think we're safe to use `h1` here, the splash page content uses `h2`

Comment (anne27) [inline client/app/app.styl]: Updated!

**Pull request merged**